blocks/article-author/article-author.js

```diff
@@ -1,11 +1,15 @@
+import ffetch from '../../scripts/ffetch.js';
 import { decorateIcons, getMetadata } from '../../scripts/lib-franklin.js';
 
-export default function decorate(block) {
+export default async function decorate(block) {
   const author = getMetadata('author');
+  const authorIndex = await ffetch('/authors/author-index.json').filter((item) => item.Name === author).first();
```

Comment:
you can skip ffetch here since we'll only need the first handful of results (ffetch is good when we need pagination).
```suggestion
  const authorIndex = await fetch('/authors/author-index.json').find((item) => item.Name === author);
```
I would probably also use `toLowerCase()` on the string comparison, or even standardize to `toClassName` so we account for minor input variations